lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -129,7 +129,7 @@ def _track_kymo(self, click, release):
 
         # Explicit copy to make modifications. Current state pushed to undo stack on assignment.
         tracks = copy(self.tracks)
-        tracks.remove_tracks_in_rect([p1, p2])
+        tracks.remove_tracks_in_rect([p1, p2], not self._adding)
```

Comment:
No, because for tracking, the behaviour has to be that it includes any that overlap partially as well. This is because the tracking cannot know about lines outside its FOV. Otherwise you can get this when tracking:

![bad_behaviour](https://user-images.githubusercontent.com/19836026/220140549-fffd4f51-6ff0-45e5-a388-48dadbd18c94.gif)

This is also the concern I expressed in the refinement, that you'd have these two behaviours. But using it now, I think this setup makes a lot of sense. Tracking includes partials, removal does not.

Reply:
ah yeah I forgot about removing tracks while doing a new tracking. yeah, this makes sense in that context then